srcpkgs/itch/files/io.itch.itch.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Type=Application
+Name=itch
+TryExec=itch
+Exec=itch %U
+Icon=itch
+Terminal=false
+Categories=Game;
+MimeType=x-scheme-handler/itchio;x-scheme-handler/itch;
+X-GNOME-Autostart-enabled=true
+Comment=Install and play itch.io games easily
```

Comment:
I greatly dislike creating our own desktop files. Does upstream not provide one at all?

Reply:
It only offers the code, no .desktop file, this code that I use to create the file.
https://github.com/itchio/itch-setup/blob/4a138757685a092640aa5fbb49c6b8133c5325cf/native/native_linux.go#L480